js/colourpicker.js

```diff
@@ -1145,7 +1115,7 @@
     function getLastColor(input) {
       if (input.data('colourpicker-lastChange')) {
         return input.data('colourpicker-lastChange');
-      } else if (input.parent().is('.palette-limited')) {
+      } else if (input.parents('.colourpicker').is('.palette-limited')) {
```

Comment:
I think `.closest()` would be a better choice, because `parents()` could potentially return multiple elements, no? We don't expect that to ever happen, but closest would make it more clear that we expect it to only match to one parent